src/ia-book-visual-adjustments.js

```diff
@@ -0,0 +1,84 @@
+import { html, LitElement } from 'lit-element';
+import { repeat } from 'lit-html/directives/repeat.js';
+import { nothing } from 'lit-html';
+import bookVisualAdjustmentsCSS from './styles/ia-book-visual-adjustments.js';
+
+export class IABookVisualAdjustments extends LitElement {
+  static get styles() {
+    return bookVisualAdjustmentsCSS;
+  }
+
+  static get properties() {
+    return {
+      options: { type: Array },
+    };
+  }
+
+  constructor() {
+    super();
+
+    this.options = [];
+  }
+
+  get activeOptions() {
+    return this.options.reduce((results, option) => (
+      option.active ? [...results, option.id] : results
+    ), []);
+  }
+
+  get activeAdjustments() {
+    const count = this.activeOptions.length;
+    return count ? html`<p>(${count} active)</p>` : nothing;
+  }
+
+  adjustmentCheckbox(option) {
+    const formID = `adjustment_${option.id}`;
+    return html`<li>
+      <label for="${formID}">
+        <span class="name">${option.name}</span>
+        <input type="checkbox" name="${formID}" id="${formID}" @change=${() => this.changeActiveStateFor(option.id)} ?checked=${option.active} />
```

Comment:
Was there discussion of an adjustment with a slider or is it just checkboxes?

Reply:
Slider variants are now in there.